### PR TITLE
Mixpanel icon

### PR DESCRIPTION
**Pull request by deepmalamittal**
mixpanel

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/7354

**Similarweb rank:** 23,003
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by sisp** [comment]
A number of unrelated commits leaked into this PR which makes the commit history in this PR a bit polluted. Could you remove them? I guess you could rebase the branch to the latest state of develop.

File: icons/mixpanel.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1 1.094c.057.053.074.11.074.187a.29.29 0 0 1-.28.283.263.263 0 0 1-.22-.113L8 11.566l-.82 1.038a.253.253 0 0 1-.221.113.29.29 0 0 1-.279-.283.24.24 0 0 1 .074-.187l.91-1.094-.89-1.078a.26.26 0 0 1-.073-.186c0-.17.11-.284.295-.284.094 0 .147.036.205.093l.779 1.017m-2.115 2.002a.313.313 0 0 1-.316-.32V9.925c0-.19.127-.32.316-.32.184 0 .315.15.315.32v2.472c0 .17-.13.32-.315.32zm0-4.356c.205 0 .369.17.369.377a.373.373 0 0 1-.37.376.377.377 0 0 1 0-.753zm-1.226 4.356a.316.316 0 0 1-.315-.32V11c0-.47-.226-.847-.726-.847s-.815.397-.815.847v1.378c0 .19-.131.32-.316.32a.316.316 0 0 1-.315-.32v-1.394c0-.474-.242-.851-.742-.851-.52 0-.779.397-.779.85v1.375c0 .19-.131.324-.315.324A.316.316 0 0 1 0 12.357V9.925c0-.19.127-.34.316-.34.22 0 .315.15.315.34v.227c.185-.36.553-.584.963-.584.484 0 .853.244 1.041.62.258-.413.648-.62 1.131-.62.816 0 1.226.584 1.226 1.395v1.434a.349.349 0 0 1-.353.32m3.64 2.246a.34.34 0 0 1-.336.34.337.337 0 0 1-.332-.34.334.334 0 1 1 .668 0m-2.693 0a.752.752 0 0 1-.742.753.753.753 0 0 1-.746-.753.75.75 0 0 1 .746-.754.75.75 0 0 1 .742.754m-3.471-.02c0 .583-.463 1.057-1.041 1.057a1.047 1.047 0 0 1-1.037-1.058c0-.583.463-1.058 1.037-1.058.578 0 1.04.475 1.04 1.058"/></svg>
```
Comment (sisp) [inline icons/mixpanel.svg:1]: I think the icon should be just the three circles without the wordmark. You can use the official `Brand Mark-Purple.svg` from the [press kit](https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9) whose link you can find near the bottom of https://mixpanel.com/news/. From looking at the press kit, I even think that the official icon isn't just three circles but a square with rounded corners and three circles inside.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -6572,6 +6572,11 @@
             "source": "https://www.mixcloud.com/about",
             "guidelines": "https://www.mixcloud.com/about"
         },
+        {
+            "title": "Mixpanel",
+            "hex": "111111",
+            "source": "https://mixpanel.com"
```
Comment (sisp) [inline _data/simple-icons.json:6578]: I think the [press kit](https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9) should be provided as a source. Unfortunately, the URL isn't very beautiful and has a different domain. :man_shrugging:

Also, please add the URL to the press kit as a brand guidelines reference.

```suggestion
            "source": "https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9",
            "guidelines:" "https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9"
```

File: _data/simple-icons.json [hunk 3]
```diff
@@ -6572,6 +6572,11 @@
             "source": "https://www.mixcloud.com/about",
             "guidelines": "https://www.mixcloud.com/about"
         },
+        {
+            "title": "Mixpanel",
+            "hex": "111111",
```
Comment (sisp) [inline _data/simple-icons.json:6577]: I think the hex value should be `#7856FF` as that's the primary color according to the [press kit](https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9).

```suggestion
            "hex": "7856FF",
```

**Review by sisp — CHANGES_REQUESTED** [review]

**Review by PratyushJayachandran — APPROVED** [review]
Perfect match 🎉 
Their website is so cool!

**Pull request merged**